EfEnumToLookup/LookupGenerator/SqlServerHandler.cs

```diff
@@ -45,59 +45,59 @@ private string BuildSql(LookupDbModel model)
 			sql.AppendLine("set nocount on;");
 			sql.AppendLine("set xact_abort on; -- rollback on error");
 			sql.AppendLine("begin tran;");
-			sql.AppendLine(CreateTables(model.Lookups));
-			sql.AppendLine(PopulateLookups(model.Lookups));
-			sql.AppendLine(AddForeignKeys(model.References));
+			sql.AppendLine(CreateTables(model));
+			sql.AppendLine(PopulateLookups(model));
+			sql.AppendLine(AddForeignKeys(model));
 			sql.AppendLine("commit;");
+
 			return sql.ToString();
 		}
 
-		private string CreateTables(IEnumerable<LookupData> enums)
+		private string CreateTables(LookupDbModel model)
 		{
 			var sql = new StringBuilder();
-			foreach (var lookup in enums)
+			foreach (var lookup in model.Lookups)
 			{
 				sql.AppendFormat(
-					@"IF OBJECT_ID('{0}', 'U') IS NULL
-begin
-	CREATE TABLE [{0}] (Id {2} CONSTRAINT PK_{0} PRIMARY KEY, Name nvarchar({1}));
-	exec sys.sp_addextendedproperty @name=N'MS_Description', @level0type=N'SCHEMA', @level0name=N'dbo', @level1type=N'TABLE',
-		@level1name=N'{0}', @value=N'Automatically generated. Contents will be overwritten on app startup. Table & contents generated by https://github.com/timabell/ef-enum-to-lookup';
-end
-",
-					TableName(lookup.Name), NameFieldLength, NumericSqlType(lookup.NumericType));
+					@"IF OBJECT_ID('{3}.{0}', 'U') IS NULL
+					begin
+						CREATE TABLE [{3}].[{0}] (Id {2} CONSTRAINT PK_{0} PRIMARY KEY, Name nvarchar({1}));
+						exec sys.sp_addextendedproperty @name=N'MS_Description', @level0type=N'SCHEMA', @level0name=N'{3}', @level1type=N'TABLE',
+							@level1name=N'{0}', @value=N'Automatically generated. Contents will be overwritten on app startup. Table & contents generated by https://github.com/timabell/ef-enum-to-lookup';
+					end{4}",
+					TableName(lookup.Name), NameFieldLength, NumericSqlType(lookup.NumericType), model.Schema, Environment.NewLine);
 			}
+
 			return sql.ToString();
 		}
 
-		private string AddForeignKeys(IEnumerable<EnumReference> refs)
+		private string AddForeignKeys(LookupDbModel model)
```

Comment:
I'm uneasy about expanding the params to be the whole model, makes it harder to see what this method actually depends on. I need to give it some more thought.